src/styles/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  width: $card-width;
+  border-radius: 5px;
+  border: 1px solid $card-border-color;
+  background: $card-background-color;
+  box-sizing: border-box;
+  padding-inline: 16px;
+  padding-bottom: 16px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    margin-top: 32px;
+    margin-inline: 3px;
+    width: $card-img-width;
+    height: $card-img-height;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    margin-top: 40px;
+    color: $card-title-color;
+    font-weight: 500;
+  }
+
+  &:hover &__title {
```

Comment:
Checklist 3

Reply:
"change the card title text color to #34568b when the card is hovered (.card:hover .card__title)"
How can i implemet this in other way?

Reply:
you can combine this selector with &:hover above